apiserver/facades/client/controller/controller.go

```diff
@@ -63,8 +70,8 @@ type ControllerAPIv3 struct {
 	*ControllerAPIv4
 }
 
-// NewControllerAPIv6 creates a new ControllerAPIv6
-func NewControllerAPIv6(ctx facade.Context) (*ControllerAPI, error) {
+// NewControllerAPIv7 creates a new ControllerAPIv7.
```

Comment:
Can we add a note about what was added between v6 and v7. I find that useful when scanning the code.

Reply:
I followed the pattern of documenting updates at the struct def level (changes are documented [here](https://github.com/juju/juju/pull/9741/files/a448a3e843c48428e57a6e854ef3fdbbab147bbb#diff-b5ab52656a34e521b266f76098f4d8e9R49))

Reply:
Ah, I missed it, sorry.